test/services/sig/index.test.ts

```diff
@@ -462,36 +464,30 @@ describe("Sig Service", () => {
 
   test("list all sigs with query", async () => {
     // Mock find and count.
-    const sigs: Sig[] = [
+    const sigs: SigBasicInfo[] = [
       {
         id: 1003,
         name: "test",
         info: "info",
         sigUrl: "url",
-        createTime: new Date("2020-08-27T03:18:51.000Z"),
         channel: "channel",
-        updateTime: new Date("2020-08-27T03:18:51.000Z"),
-        status: 0,
-        lgtm: 2,
+        membersCount: 7,
+        needsLGTM: 2,
       },
       {
         id: 1004,
         name: "test1",
         info: "info",
         sigUrl: "url",
-        createTime: new Date("2020-08-27T03:18:51.000Z"),
         channel: "channel",
-        updateTime: new Date("2020-08-27T03:18:51.000Z"),
-        status: 0,
-        lgtm: 2,
+        membersCount: 7,
+        needsLGTM: 2,
       },
     ];
-    const sigFindOneMock = jest.spyOn(sigRepository, "findAndCount");
-    sigFindOneMock.mockImplementation((options) => {
-      // @ts-ignore
-      const { skip, take } = options;
 
-      return Promise.resolve([sigs.slice(skip, take), sigs.length]);
+    const sigFindOneMock = jest.spyOn(sigRepository, "listSigsAndCount");
```

Comment:
There seems to be an inconsistency between the variable name and the function of mock.

Reply:
Fixed.